substreams/ethereum-rocketpool/src/modules/2_map_relative_component_balance.rs

```diff
@@ -0,0 +1,55 @@
+use crate::{
+    abi::rocket_deposit_pool,
+    constants::{ETH_ADDRESS, ROCKET_DEPOSIT_POOL_ADDRESSES, ROCKET_POOL_COMPONENT_ID},
+};
+use anyhow::Result;
+use substreams_ethereum::{pb::eth, Event};
+use tycho_substreams::models::{BalanceDelta, BlockBalanceDeltas};
+
+/// Extracts ETH balance changes for rocket pool component
+///
+/// This function uses RocketDepositPool events to extract ETH balance changes. If a
+/// deposit to the component is detected, it's balanced is increased and if a balance
+/// from the component is withdrawn its balance is decreased.
+#[substreams::handlers::map]
+fn map_relative_component_balance(block: eth::v2::Block) -> Result<BlockBalanceDeltas> {
```

Comment:
Technically this is tracking available liquidity on the deposit pool and isn't tracking the total protocol balance. The values from here should be emitted as the `liquidity` attribute and not as the component's balance. Let's rename this to make it clearer?
```suggestion
fn map_relative_liquidity(block: eth::v2::Block) -> Result<BlockBalanceDeltas> {
```